sdk/contentsafety/azure-ai-contentsafety/README.md

```diff
@@ -0,0 +1,292 @@
+
+### Analyze text
+
+#### Analyze text without blocklists
+```java com.azure.ai.contentsafety.analyzetext
+String endpoint = Configuration.getGlobalConfiguration().get("CONTENT_SAFETY_ENDPOINT");
+String key = Configuration.getGlobalConfiguration().get("CONTENT_SAFETY_KEY");
+ContentSafetyClient contentSafetyClient = new ContentSafetyClientBuilder()
+    .credential(new AzureKeyCredential(key))
+    .endpoint(endpoint).buildClient();
+
+AnalyzeTextResult response;
+try {
+    response = contentSafetyClient.analyzeText(new AnalyzeTextOptions("This is text example"));
+} catch (HttpResponseException ex) {
+    System.out.println(String.format("Analyze text failed.\nStatus code: %s, Error message: %s",
+        ex.getResponse().getStatusCode(), ex.getMessage()));
+    throw ex;
+}
+System.out.println("Hate severity: " + response.getHateResult().getSeverity());
+System.out.println("SelfHarm severity: " + response.getSelfHarmResult().getSeverity());
+System.out.println("Sexual severity: " + response.getSexualResult().getSeverity());
+System.out.println("Violence severity: " + response.getViolenceResult().getSeverity());
+```
+
+#### Analyze text with blocklists
+```java com.azure.ai.contentsafety.analyzetextwithblocklist
+// After you edit your blocklist, it usually takes effect in 5 minutes, please wait some time before analyzing with blocklist after editing.
+AnalyzeTextOptions request = new AnalyzeTextOptions("I h*te you and I want to k*ll you");
+request.getBlocklistNames().add(blocklistName);
+request.setBreakByBlocklists(true);
+
+AnalyzeTextResult analyzeTextResult;
+try {
+    analyzeTextResult = contentSafetyClient.analyzeText(request);
+} catch (HttpResponseException ex) {
+    System.out.println("Analyze text failed.\nStatus code: " + ex.getResponse().getStatusCode() + ", Error message: " + ex.getMessage());
+    throw ex;
+}
+
+if (analyzeTextResult.getBlocklistsMatchResults() != null) {
+    System.out.println("\nBlocklist match result:");
+    for (TextBlocklistMatchResult matchResult : analyzeTextResult.getBlocklistsMatchResults()) {
+        System.out.println("Blockitem was hit in text: Offset: " + matchResult.getOffset() + ", Length: " + matchResult.getLength());
+        System.out.println("BlocklistName: " + matchResult.getBlocklistName() + ", BlockItemId: " + matchResult.getBlockItemId() + ", BlockItemText: " + matchResult.getBlockItemText());
+    }
+}
+```
+
+### Analyze image
+```java com.azure.ai.contentsafety.analyzeimage
+String endpoint = Configuration.getGlobalConfiguration().get("CONTENT_SAFETY_ENDPOINT");
+String key = Configuration.getGlobalConfiguration().get("CONTENT_SAFETY_KEY");
+
+ContentSafetyClient contentSafetyClient = new ContentSafetyClientBuilder()
+    .credential(new AzureKeyCredential(key))
+    .endpoint(endpoint).buildClient();
+
+ImageData image = new ImageData();
+String cwd = System.getProperty("user.dir");
+String absolutePath = cwd + "./src/image.jpg";
```

Comment:
resource should be under subfolder of samples or test folder (definitely not directly in src folder).

Prefer to be under `src/test/resources`
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileShareTestBase.java#L95

@srnagar Do we have good example of loading file in sample? storage-file-share probably not a good example https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/storage/azure-storage-file-share/src/samples/java/com/azure/storage/file/share/FileSample.java#L93-L94

Reply:
Wait for Srikanta's comment.

My opinion is that in test it should be `getClassLoader().getResource()`.

Not sure about sample. But if we use file, one should use `Paths.get()` to concat file path.